app/scripts/main.js

```diff
@@ -1,9 +1,11 @@
 $(function () {
   let pricingData = [];
-
+  let sortOrder = false; // false means the sorting will be in Increasing order and true will be vice versa
```

Comment:
I'm not happy with the naming here.

Tip: good code is self explanatory. If you had to comment the working, then it needs improvement